app/src/components/map/FeaturePopup.tsx

```diff
@@ -58,47 +102,66 @@ const useStyles = makeStyles(() => ({
   }
 }));
 
-const FeaturePopup: React.FC<React.PropsWithChildren<{ submissionSpatialComponentId: number }>> = (props) => {
-  const { submissionSpatialComponentId } = props;
+const FeaturePopup: React.FC<React.PropsWithChildren<{ submissionSpatialComponentIds: number[] }>> = (props) => {
+  const { submissionSpatialComponentIds } = props;
 
   const classes = useStyles();
   const api = useApi();
+  const [currentIndex, setCurrentIndex] = useState(0);
 
   const dataLoader = useDataLoader(() => {
```

Comment:
For future PRs: we should use a descriptive name for this const, something like `spatialDataLoader` or something.  Just so it is a bit more self-documenting when its used further down the code.